sopel/tools/__init__.py

```diff
@@ -15,20 +15,15 @@
 
 import codecs
 from collections import defaultdict
-import functools
-import inspect
 import logging
 import os
 import re
 import sys
 import threading
-import traceback
-
-from pkg_resources import parse_version
-
-from sopel import __version__
 
+from sopel.deprecated import deprecated
```

Comment:
Forgot to add this before submitting the review. 😑

```suggestion
from sopel.deprecated import deprecated  # Don't delete; maintains backward compatibility with pre-8.0 API
```